config/data/paragonUtils.js

```diff
@@ -0,0 +1,160 @@
+const path = require('path');
+const fs = require('fs');
+
+/**
+ * Attempts to extract the Paragon version from the `node_modules` of
+ * the consuming application.
+ *
+ * @param {string} dir Path to directory containing `node_modules`.
+ * @returns {string} Paragon dependency version of the consuming application
+ */
+function getParagonVersion(dir, { isBrandOverride = false } = {}) {
+  const npmPackageName = isBrandOverride ? '@openedx/brand' : '@openedx/paragon';
+  const pathToPackageJson = `${dir}/node_modules/${npmPackageName}/package.json`;
+  if (!fs.existsSync(pathToPackageJson)) {
+    return undefined;
+  }
+  return JSON.parse(fs.readFileSync(pathToPackageJson, 'utf-8')).version;
+}
+
+/**
+ * @typedef {Object} ParagonThemeCssAsset
+ * @property {string} filePath
+ * @property {string} entryName
+ * @property {string} outputChunkName
+ */
+
+/**
+ * @typedef {Object} ParagonThemeVariantCssAsset
+ * @property {string} filePath
+ * @property {string} entryName
+ * @property {string} outputChunkName
+ */
+
+/**
+ * @typedef {Object} ParagonThemeCss
+ * @property {ParagonThemeCssAsset} core The metadata about the core Paragon theme CSS
+ * @property {Object.<string, ParagonThemeVariantCssAsset>} variants A collection of theme variants.
+ */
+
+/**
+ * Attempts to extract the Paragon theme CSS from the locally installed `@openedx/paragon` package.
+ * @param {string} dir Path to directory containing `node_modules`.
+ * @param {boolean} isBrandOverride
+ * @returns {ParagonThemeCss}
+ */
+function getParagonThemeCss(dir, { isBrandOverride = false } = {}) {
+  const npmPackageName = isBrandOverride ? '@openedx/brand' : '@openedx/paragon';
```

Comment:
The reason why `@edx/brand` is not considered is that we would like to encourage the migration of that alias as much as possible ([additional context](https://openedx.slack.com/archives/C071U9E4VNV/p1715940913033459)), that is why is included in the list of changes in the PR description,  as you can see in the [example app PR](https://github.com/openedx/frontend-build/pull/549/files#diff-a593fc14fdef41aaf092f774b92cb74bafeda3e9559f8ae4762374bf4af80c92R18) 

Even the current readme for the repository makes reference to `@openedx/brand` instead of `@edx/brand`.

https://github.com/openedx/frontend-build/blob/ae8ce98d5252d9ddd4e730d6b21ca40257453bef/README.md?plain=1#L158

I consider that we could be more explicit about this change in the [MFE Migration Guide](https://openedx.atlassian.net/wiki/spaces/BPL/pages/3770744958/Migrating+MFEs+to+Paragon+design+tokens+and+CSS+variables) instead. What do you think?

Reply:
> Even the current readme for the repository makes reference to @openedx/brand instead of @edx/brand.

To expand on @brian-smith-tcril's response below regarding keeping parity with the existing `@edx/brand` scope used today given the non-breaking nature of this release, I believe the README prematurely referenced `@openedx/brand` as part of [this PR](https://github.com/openedx/frontend-build/pull/490) given that migration of NPM scope for the brand packages has not yet been prioritized/executed. Looks like the brand package was incorrectly grouped with the scope migrations for other NPM packages (i.e., `@openedx/paragon` and `@openedx/frontend-build`).

By having the current README recommend `@openedx/brand` in the `module.config.js` within the README would never be resolved as the alias since `@openedx/brand` is not used in consuming apps, so it's currently incorrect documentation.